manager/allocator/allocator_test.go

```diff
@@ -127,14 +133,15 @@ func TestAllocator(t *testing.T) {
 				State: api.TaskStateNew,
 			},
 			ServiceID:    "testServiceID1",
+			Spec:         s1.Spec.Task,
 			DesiredState: api.TaskStateRunning,
 		}
 		assert.NoError(t, store.CreateTask(tx, t2))
 
 		// Create the predefined node-local network with one service
-		assert.NoError(t, store.CreateNetwork(tx, p))
+		// assert.NoError(t, store.CreateNetwork(tx, p))
```

Comment:
Would be cleaner to actually remove all of these instead of just commenting out?